scripts/generate-app-config/generatePagesConfig.js

```diff
@@ -0,0 +1,240 @@
+const path = require('path');
+const glob = require('glob');
+const kebabCase = require('lodash.kebabcase');
+const lodashStartCase = require('lodash.startcase');
+
+/**
+* Cheat. If the filename still contains a period, don't run startcase. This allows for filenames of version (v0.5.0)
+*/
+const startCase = (string) => {
+  if (string.includes('.')) {
+    return string;
+  }
+  return lodashStartCase(string);
+};
+
+/**
+* Gathers the complete set of requested page types.
+*/
+const pageTypes = navConfig => (navConfig.navigation.links.reduce((acc, link) => acc.concat(link.pageTypes), []));
+
+/**
+* Gets the path relative to the dev-site-config directory.
+*/
+const relativePath = componentPath => (path.relative(path.join(process.cwd(), 'dev-site-config'), componentPath));
```

Comment:
Does this need to be rootPath, not processPath to correctly generate the app config for package-level tests?
```
  const rootPath = processPath.includes('packages') ? processPath.split('packages')[0] : processPath;
```

Reply:
nope, we build a subset site, just for the package being tested.

Reply:
oh cool!